cmd/juju/controller/kill.go

```diff
@@ -184,6 +184,13 @@ func (c *killCommand) DirectDestroyRemaining(ctx *cmd.Context, api destroyContro
 		hasErrors = true
 		logger.Errorf("unable to retrieve hosted model config: %v", err)
 	}
+	ctrlUUID := ""
+	// try to get controller UUID or just ignore.
+	if ctrlCfg, err := api.ControllerConfig(); err == nil {
+		ctrlUUID = ctrlCfg.ControllerUUID()
+	} else {
+		logger.Errorf("getting controller config from API: %v", err)
```

Comment:
why can we ignore this error?
if we can't do anything about it it needs to be a warning not an error

Reply:
changed to `Warningf`